### DISCUSSION TITLE
ScalarBar cannot be displayed on window 7

### DISCUSSION THREAD

**Discussion by chenjiajun1991** [Kitware/trame#96] [Q&A]
The same set of code, scalar can not be displayed on window 7, but can be displayed normally on window10, mac。
<img width="357" alt="1657875058(1)" src="https://user-images.githubusercontent.com/17824508/179189027-7d458b52-ebb4-4f0d-bdbc-92121e660491.png">

Displaying on window10 is like this, and displaying scalar on window 7 is not displayed.
![1657875315(1)](https://user-images.githubusercontent.com/17824508/179190365-89a7fe99-37a8-45ea-bfe4-a88ead4919ba.jpg)

**Comment by jourdain** [comment]
Which version of your browser is getting used on windows 10 vs 7 ?

**Reply by pazars** [reply]
Same problem on Windows 10 - has the NaN and misses max value. Happens irrespective of browser.
The vtkScalarBarWidget doesn't work properly. The issue persists even when there's just a scalar bar actor.
Tried Firefox 102.0.1 (64-bit) and Chrome 103.0.5060.134 (Official Build) (64-bit)

**Comment by jourdain** [comment]
Seems to be related to state serialization of scalar bar

**Reply by jourdain** [reply]
https://github.com/Kitware/trame/issues/101
